apps/condo/domains/ticket/schema/TicketReportService.test.js

```diff
@@ -25,16 +21,16 @@ describe('TicketReportService', () => {
         })
 
         it('can not get ticket report with another organization', async () => {
```

Comment:
please add two tests for two cases: user without role and user with role

Reply:
Done